examples/profile_scrape/dynamic_profile.py

```diff
@@ -18,19 +20,22 @@
 class DynamicProfile(Profile):
     """Subclass of Profile to provide some dynamic functionality using Selenium"""
 
-    def dynamic_load(self):
+    def dynamic_load(self, max_posts=sys.maxsize):
         """Scroll the users Profile page and load all posts into Post objects"""
         source_data, browser = self._scroll_page()
         post_soup = self._seperate_posts(source_data)
-        self._create_post_objects(post_soup)
+        self._create_post_objects(post_soup, max_posts)
         self._grab_useful_data()
         browser.close()
 
     def _scroll_page(self):
         """Scroll page and load the source code"""
-        browser = Chrome(
-            r"C:\Users\Chris\chromedriver.exe", chrome_options=chrome_options
-        )
+        root_dir = Path(__file__).parent.parent.parent
+        if platform == "linux" or platform == "linux2" or platform == 'darwin':
+            path = 'chromedrivers'
+        else:
+            path = 'win.exe'
+        browser = Chrome(os.path.join(root_dir, 'chromedrivers/{}'.format(path)), options=chrome_options)
```

Comment:
lol definitely a good idea generalizing this, I honestly have no idea what I was thinking leaving my chromedriver path for the user to have to change 🤣  

Instead of having the chromedrivers in the repo though, do you think you could yank those files out and leave it up to the user to pass their own browser object or `'/path/to/driver.exe'`? I figure for maintainability (and security) we can leave it up to them to either
- pass the filepath to their chromedriver with something like 
```python
def dynamic_load(self, executable_path, max_posts=sys.maxsize)
```
where `executable_path` is the path to their chromedriver  
or alternatively (and perhaps preferably)
- pass their own browser object with something like 
```python
def dynamic_load(self, browser, max_posts=sys.maxsize)
```
where `browser` is a Selenium webdriver of their choice

Either one works for me so that's totally up to you with what you decide (or if you think of a better alternative)! 
If this is confusing or you don't feel like changing it tho then no worries lol I don't mind making the change

Reply:
I don't know if there could be some security issue with that but, as you asked, I'm going to implement the suggestions :D I thought it could be helpful have the drivers directly in the project for those who run the test to understand the potentiality of the library :) I prefer passing the browser preference directly :D